src/lib/converter/plugins/DynamicModulePlugin.ts

```diff
@@ -23,6 +23,9 @@ export class DynamicModulePlugin extends ConverterComponent {
      */
     private reflections!: Reflection[];
 
+    private hiddenExtension = ['.ts', '.d'];
```

Comment:
Did .ts extensions show up somewhere? I've never seen them...

Reply:
Ah, I see this is because it now overwrites the name... not sure about this. Will have to play with it.

Reply:
The goal is to avoid the remaining.d at the end of the file.
It supports the .ts, the .d.ts and classic cases (.js, .jsx, .tsx,...) Thanks to the extension.
But extension alone is not working for .d.ts